src/ui/shared/Replies.tsx

```diff
@@ -73,6 +73,7 @@ const Replies: React.FC<Props> = ({
 
 				return (
 					<Reply
+						key={reply.id}
```

Comment:
This prop was added to avoid the following error:
![image](https://user-images.githubusercontent.com/36703917/167375596-f31716b0-a2d8-4415-ad11-79ef3559a239.png)